hudi-common/src/main/java/org/apache/hudi/common/util/AvroOrcUtils.java

```diff
@@ -796,4 +800,32 @@ private static Schema getActualSchemaType(Schema unionSchema) {
       return Schema.createUnion(nonNullMembers);
     }
   }
+
+  public static void addAvroRecord(
```

Comment:
can this sit somewhere in a test utils class? given its only used by a test?

Reply:
Sure, Done.